src/envo/envo-edit.owl

```diff
@@ -46308,6 +46317,86 @@ AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibra
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03620012> "landscape mesoscale age"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03620012> <http://purl.obolibrary.org/obo/PATO_0001486>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03620013> (landscape microscale age)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03620013> "Chronological age which inheres in a bearer by virtue of that bearer existing for between1 and 500 years.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_03620013> "https://orcid.org/0000-0002-4366-3088")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_03620013> <https://orcid.org/0000-0002-1296-2528>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_03620013> "2024-12-16T14:35:46Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.obolibrary.org/obo/ENVO_03620013> "https://en.wikipedia.org/wiki/Scale_(geography)")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03620013> "landscape microscale age"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03620013> <http://purl.obolibrary.org/obo/PATO_0001486>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03620014> (landscape megascale size)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03620014> "An area quality which inheres in a bearer by virtue of that bearer having an area greater than 10,000,000,000,000 square metres")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_03620014> "https://orcid.org/0000-0002-4366-3088")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_03620014> <https://orcid.org/0000-0002-1296-2528>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_03620014> "2024-12-16T14:43:20Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.obolibrary.org/obo/ENVO_03620014> "https://en.wikipedia.org/wiki/Scale_(geography)")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03620014> "landscape megascale size"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03620014> <http://purl.obolibrary.org/obo/PATO_0001323>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03620015> (landscape macroscale size)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03620015> "An area quality which inheres in a bearer by virtue of that bearer having an area of between 10,000,000,000 and 10,000,000,000,000 square metres")
```

Comment:
10e10 is 100,000,000,000

Reply:
I prefer exponential notation, but is it even true that this is an accepted cutoff? Sources would be good